docs/operator-docs/kinesis-stream-for-ingest.md

```diff
@@ -0,0 +1,26 @@
+---
+id: kinesis-stream-for-ingest
+title: Kinesis Stream For Ingest
+hide_title: true
+---
+
+# Setup Kinesis Stream & CNM Message
+
+> **Note**: Keep in mind that you should only have to set this up once. This should be created via the SDS or from a cloud formation template. Kinesis pricing is based on the shard value and not on amount of kinesis usage.
+
+1. Create a Kinesis Stream
+
+* Go to the `Kinesis` service and click `Create Data Stream`.
```

Comment:
```suggestion
* Go to the `Kinesis` service in your AWS console and click `Create Data Stream`.
```

Reply:
We can reuse the screenshot from here: https://nasa.github.io/cumulus/docs/data-cookbooks/cnm-workflow#kinesis

Reply:
@laurenfrederick Added the screenshot per suggestion.